templates/explore/user_list.tmpl

```diff
@@ -26,6 +26,12 @@
 			</div>
 		</div>
 	{{else}}
-		<div class="flex-item">{{ctx.Locale.Tr "explore.user_no_results"}}</div>
+		<div class="flex-item">
+		{{if .PageIsExploreUsers}}
+			{{ctx.Locale.Tr "search.user_no_results"}}
+		{{else}}
+			{{ctx.Locale.Tr "search.org_no_results"}}
+		{{end}}
+		</div>
```

Comment:
Could it be a general "search.no_results" ?

Reply:
Or just `no_results` even. We need to cut down on unnecessary translation namespacing.